convert2rhel/actions/system_checks/duplicate_packages.py

```diff
@@ -31,10 +32,17 @@ def run(self):
         super(DuplicatePackages, self).run()
 
         logger.task("Prepare: Check if there are any duplicate installed packages on the system")
-        output, _ = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
+        output, ret_code = utils.run_subprocess(["/usr/bin/package-cleanup", "--dupes", "--quiet"], print_output=False)
         if not output:
             return
-
+        # For el7 machines we have to depend on this output to know the check failed
```

Comment:
Maybe I would add comment, which type of error we are capturing by the if chain. Something like:
```suggestion 
        # Catching situation when repositories cannot be accessed.
        # The output from the package-cleanup is: [Errno -2] Name or service not known
        # For el7 machines we have to depend on this output to know the check failed
```

Reply:
Sounds good!